flink-runtime/src/main/java/org/apache/flink/runtime/memory/MemoryManager.java

```diff
@@ -89,12 +89,14 @@
 	 *
 	 * @param memorySize The total size of the off-heap memory to be managed by this memory manager.
 	 * @param pageSize The size of the pages handed out by the memory manager.
+	 * @param verifyEmptyWaitGcMaxSleeps defines how long to wait for GC of all allocated memory to check for memory leaks,
+	 *                                   see also {@link UnsafeMemoryBudget}.
```

Comment:
Hmm, the description says that it is the time to wait for GC but isn't is the number of exponential retries?

Reply:
You are right, I just did not want to repeat all details here, so I referred to `UnsafeMemoryBudget` for full description.